### ISSUE TITLE
Clarify relationship with webpack-serve

### ISSUE THREAD

**Issue by kadamwhite** [webpack/webpack-dev-server#1493]
I've been noticing some back-and-forth in the README that makes it unclear what the future of this project is, or whether webpack-contrib/webpack-serve is a successor to this project or not. It would be tremendously helpful to the community at large if the README included a statement about the relationship between the two projects, because I've seen many conflicting statements from dev-server and Webpack maintainers in project READMEs, issue thread, and on twitter.

Please advise; and thank you all for your work building excellent development tools :)

**Comment by michael-ciniawsky** [comment]
**TL:DR** Use `weback-dev-server`, it's actively maintained

It isn't really decided yet, from a user point of view use `webpack-dev-server` for now, since it isn't possible to maintain `webpack-serve` atm. It's original author abandoned `webpack-serve` and some it's dependencies are exclusively owned by him. I would co-maintain both packages until a ROADMAP is settled, but it would require an unjustifiable amount of additional work and resources to resolve the aforementioned dependency locks in `webpack-serve`. The rough plans currently are either to 

1. Modernize `webpack-dev-server` similiar to `webpack-serve` (http2 (`koa`), websockets (`ws`))(`webpack-dev-server` v4.0.0) 
2. Or if and only if the `webpack-cli` gets support for subcommands (e.g `webpack serve --options`) in the near future to eventually reused at least the package **name** (`webpack-serve`) for naming consistency. This may involve other (bigger) refactorings aswell then

**Comment by michael-ciniawsky** [comment]
To clarify 1. is more likely to happen as of today but **definitely not** finally atm

**Comment by evilebottnawi** [comment]
@kadamwhite Briefly: use `webpack-dev-server` (`webpack-serve` will be archived in near future)
@michael-ciniawsky we can't use `webpack-serve` name due Andrew did not give us consent to this (he is owner and can do with package anything)

**Comment by kadamwhite** [comment]
Thank you both. I feel some README note might be beneficial here, given that less than a month ago I was hearing otherwise; however I understand if the situation with `webpack-serve` might preclude that. I appreciate your responses regardless.

**Comment by jeremejevs** [comment]
@michael-ciniawsky 

> It's original author abandoned `webpack-serve`

Source? His [latest commit](https://github.com/shellscape/webpack-serve/commit/c5b1a1d19841e9932763210cd31e8ef175a01a0c) suggests otherwise. The tone also implies conflict, what happened?

**Comment by evilebottnawi** [comment]
@jeremejevs it is fork, `webpack` and `webpack-contrib` don't support `webpack-serve` officially, you can use it at your own risk. Some developers create problems, not code. Closer to the new year we will try to release a new version - lightweight and using `websocket`.

**Comment by michael-ciniawsky** [comment]
Besides of removing the deprecation note I don't see any changes being made to this fork? If the original author intends to keep maintaining `webpack-serve` and it's dependencies, he can of course do that as he pleases, but from an organizational stand point it's not possible for other contribs to do the same since some dependencies of `webpack-serve` are exclusively owned by him and would need to be forked/republished which we decided against. e.g `webpack-serve` and co could be moved to his account by commiting a `LICENSE` change and requesting a transfer at any time. I wasn't involved in any conflict around this and neither do I have a preference or 'strong feeling'  about that matter. The main issue relies in the unnecessary confusion for users and the lack of feasible options for other contribs to solve it e.g we may have continued 'promoting'  `webpack-serve` as an successor of `webpack-dev-server` since the code is definitely more future proofen then the current code base of `webpack-dev-server`, but again this isn't possible

**Comment by evilebottnawi** [comment]
The question is answered. Still use `webpack-dev-server` (`webpack-server` is not supported officially and you use this package at one's own risk). Feel free to feedback.
